storage/rocksdb.go

```diff
@@ -222,17 +223,27 @@ func (store *rocksStore) init() error {
 	return nil
 }
 
+var (
+	VALID_QUEUE_NAME = regexp.MustCompile(`^[a-zA-Z0-9._-]*$`)
+)
+
 func (store *rocksStore) GetQueue(name string) (Queue, error) {
 	if name == "" {
-		return nil, fmt.Errorf("Queue name cannot be blank")
+		return nil, fmt.Errorf("queue name cannot be blank")
 	}
+
 	store.mu.Lock()
 	defer store.mu.Unlock()
 
 	q, ok := store.queueSet[name]
 	if ok {
 		return q, nil
 	}
+
+	if ! VALID_QUEUE_NAME.MatchString(name) {
+		return nil, fmt.Errorf("queue names can only contain A-Za-z0-9_.-")
```

Comment:
Might be better to include the regexp so we don't have to update the error message if we relax the regexp, i.e.:

fmt.Errorf("queue name must match %v", ValidQueueName)

Reply:
I'm not sure how Regexps format as strings so this may not be a good suggestion.  :-)